src/components/legend/draw.js

```diff
@@ -63,6 +63,19 @@ module.exports = function draw(gd) {
         .call(Color.fill, opts.bgcolor)
         .style('stroke-width', opts.borderwidth + 'px');
 
+
+    var title = fullLayout.legend.title;
+    gd._fullLayout._legendTitleWidth = 0;
+    gd._fullLayout._legendTitleHeight = 0;
```

Comment:
We define `var opts = fullLayout.legend` above, so this block should be:

```js
var title = opts.title;
opts._legendTitleWidth = 0;
opts._legendTitleHeight = 0;
```

Reply:
Good call. 
Addressed in https://github.com/plotly/plotly.js/pull/4386/commits/a9759a27029ce39dd1071d570b8254d0ea667d00.